src/main/kotlin/Actions/GetHTMLResponseAction.kt

```diff
@@ -0,0 +1,23 @@
+package Actions
+
+import httpstatus.HttpStatus
+import response.ResponseBuilder
+
+class GetHTMLResponseAction : Action {
+    private val statusCode = HttpStatus.OK
+    private val headers = mapOf("Content-Type" to "text/html;charset=utf-8")
```

Comment:
I wonder if you could put this in an enum or a map. So, instead of having to specify `"text/html;charset=utf-8"`, you could do `ContentType.HTML` (or something like that). 

I think when I did the HTTP Server, I used a third party library that had all the MIME types. MIME types might be a good thing to read a bit on if you haven't. https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types

Reply:
I can do something similar to `enum` `HttpStatus`. Thanks, I'll look into it a bit more.